jaeger-core/src/test/java/io/jaegertracing/internal/senders/SenderResolverTest.java

```diff
@@ -81,6 +81,22 @@ public void testMultipleImplementationsAmbiguous() throws Exception {
     assertTrue(sender instanceof NoopSender);
   }
 
+  @Test
+  public void testMultipleFactoriesButFactoryNotSpecified() throws Exception {
```

Comment:
Isn't this the same as `testMultipleImplementationsAmbiguous` ?

Reply:
True. My bad. Removed redundant test.